scripts/k8s/postgres.yaml

```diff
@@ -24,7 +24,7 @@ spec:
       - env:
         - name: POSTGRES_PASSWORD
           value: password
-        image: postgres:9
+        image: cargo.caicloud.io/caicloud/postgres:9
```

Comment:
By the way, could you update the docker-compose.yml and docker-compose-dev.yml?

:)